packages/components/pie-textarea/src/index.ts

```diff
@@ -213,8 +224,19 @@ export class PieTextarea extends FormControlMixin(RtlMixin(LitElement)) implemen
                     ?disabled=${disabled}
                     @input=${this.handleInput}
                     @change=${this.handleChange}
+                    aria-describedby=${ifDefined(assistiveText ? assistiveTextIdValue : undefined)}
+                    aria-invalid=${status === 'error' ? 'true' : 'false'}
+                    aria-errormessage="${ifDefined(status === 'error' ? assistiveTextIdValue : undefined)}"
                 ></textarea>
-            </div>`;
+            </div>
+            ${assistiveText ? html`
+                <pie-assistive-text
+                    id="${assistiveTextIdValue}"
+                    variant=${ifDefined(status)}
+                    data-test-id="pie-textarea-assistive-text">
+                    ${assistiveText}
+                </pie-assistive-text>
+            ` : nothing}`;
```

Comment:
I think it could be cleaner to move this to a function named `renderAssistiveText`. This function can contain the conditional logic and return the template or `nothing`. Then this line is just `this.renderAssistiveText()`.

We have done similar in other components for simplifying templates.